scripts/tier_analyzer.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+import argparse
+import re
+import subprocess
+import sys
+
+
+def check_virtualenv():
+    """Check if we are in virtualenv and if not, exit."""
+    if not hasattr(sys, 'real_prefix'):
+        print('You must activate CFME virtualenv in oder to run this script.')
+        sys.exit(1)
+
+
+def get_pytest_output(args, use_tier_marker=False):
+    """Get string output of pytest command."""
```

Comment:
**REQUIRED:** The docstring needs to be more specific this is not just generic output of just any pytest command.   It is pytest --collect-only output...and specifically for the purpose of finding out what tests would run for the specified provider.   All this information need to be included in the